npm-distribution/yarn.lock

```diff
@@ -253,28 +253,28 @@ string-width@^5.0.1, string-width@^5.1.2:
 
 "strip-ansi-cjs@npm:strip-ansi@^6.0.1":
   version "6.0.1"
-  resolved "https://registry.yarnpkg.com/strip-ansi/-/strip-ansi-6.0.1.tgz#9e26c63d30f53443e9489495b2105d37b67a85d9"
```

Comment:
should you be using yarn instead of npm here? Or is this just a change in the yarn tool?